### DISCUSSION TITLE
Gyro calibration new beta

### DISCUSSION THREAD

**Discussion by Kermit647** [pybricks/support#1080] [General]
Feedback part :
I have two Spike Prime bricks.

One will pass the gyro calibration just barely.
It will report stationary, but just tapping the table lightely with the finger will make it not stationary.

The other one will never report stationary and never pass imu.ready().

( I am aware you can change the sensitivity of the stationary check)

Worries part :

Has anyone ever use this on a competition ?
We will be at the WRO german finals at 16. Juni and intend to use it.

Without bluetooth connected, the hub switches off pretty fast. I am worried we will not get the gyro calibrated in a crowded hall on a wanky table with kids and jury running and bumping around.

**Comment by laurensvalk** [comment]
That is indeed what the sensitivities are for.

Maybe it's better if we use less sensitive values by default.

What do you think?

**Reply by Kermit647** [reply]
I made some tests with the two Prime Hubs I have available.

I think the accelaration threashold can be completely left out. This is the one that is most sensible to vibrations and shock. Trampling on the ground or tapping the table will trigger it, even if you set relatively high values. I don't think it is neccesary to check this value at all. Theoreticalle the gyro can be calibrated anyway no matter what force values the robot it is experiencing as it is only the rotation that matters.

For the gyro I think a setting of about 5 will be ok. With this setting it is impossible to trigger steady when holding thy robot in the hand or when the robot is driving around on the table. With a setting of 10 it is possible the robot will sometimes go to steady state when doing a very calm straight drive.

**Reply by laurensvalk** [reply]
Thanks for the input!
